.github/workflows/ci.yml

```diff
@@ -52,6 +76,14 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
 
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v35
+        with:
+          files: |
+            docs/**
+            packages/create-vite/**
+
```

Comment:
```suggestion
```
I think we can remove this part as it's extracted into a different job.

Reply:
True forgot to cleanup